source/blog/2015-03-01-cappuccino-news-2015-03-01.markdown

```diff
@@ -0,0 +1,74 @@
+
+- The method `setNeedsLayout:` and `needsLayout` has been added to the class `CPView`. You can now easily control to lay out or not a view. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2316))
+
+- `Cappuccino` now supports the `KVC` operators `unionOfObjects`, `distinctUnionOfObjects`, `unionOfArrays`, `distinctUnionOfArrays` and `distinctUnionOfSets`. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2313))
+
+- The method `viewAtColumn:row:makeIfNecessary` has been added to the class `CPTableView`. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2315))
+
+##### Changes/Fixes
+
+- The method `setValue:forKey:` of the `CPObjectController` does not use the `theKeyPath` parameter as `keyPath` and `value` anymore. ([krodelin](https://github.com/krodelin) - [PR](https://github.com/cappuccino/cappuccino/pull/2301))
+
+- Changed the name of the method `CPDomDisplayServerSetStyleBackgroundSize` to `CPDOMDisplayServerSetStyleBackgroundSize`. ([ahankinson](https://github.com/ahankinson) - [PR](https://github.com/cappuccino/cappuccino/pull/2296))
+
+- Removed unreachable code in `CPDOMDisplayServer.h`. ([ahankinson](https://github.com/ahankinson) - [PR](https://github.com/cappuccino/cappuccino/pull/2300))
+
+- The `CPStepper` class method returns `id` instead of `CPStepper` type. ([ahankinson](https://github.com/ahankinson) - [PR](https://github.com/cappuccino/cappuccino/pull/2294))
+
+- The `CPTableView` class has been refactored and optimized. The methods `preparedViewAtColumn:row` and `enumerateAvailableViewsUsingblock:` have been added. ([cacaodev](https://github.com/cacaodev) - [PR](https://github.com/cappuccino/cappuccino/pull/1892) [PR](https://github.com/cappuccino/cappuccino/pull/2303) [PR](https://github.com/cappuccino/cappuccino/pull/2307) [PR](https://github.com/cappuccino/cappuccino/pull/2306) [PR](https://github.com/cappuccino/cappuccino/pull/2312) [PR](https://github.com/cappuccino/cappuccino/pull/2318))
+
+- The class `BKShowcaseController` of the framework `BlendKit` has been fixed. The showcase of a theme works again. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2295))
+
+- Duplicated unit tests have been removed or renamed. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2305))
+
+- The `CPDatePicker` can not take an `invalid javascript date` anymore. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2308))
+
+- The divider color of a `CPSegmentedControl` now has the expected color. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2309))
+
+- The `CPPopover` updates its position even if a frame of one of its superviews has changed. ([Dogild](https://github.com/Dogild) - [PR](https://github.com/cappuccino/cappuccino/pull/2311))
```

Comment:
\`CPPopover\` updates its position when the frame of one of its superviews has changed.